js/utils/DateUtils.js

```diff
@@ -0,0 +1,10 @@
+const moment = require('moment')
```

Comment:
As was discussed in our chat, moment is not good library to use (I was  affected as well and changed it to date-fns)